Mono.Addins/Mono.Addins.Database/SetupProcess.cs

```diff
@@ -187,10 +178,8 @@ static string CreateHostExe ()
 			ilg.EmitCall (OpCodes.Call, mi, null);
 			ilg.Emit (OpCodes.Ret);
 			
-			tb.CreateType();
-			ab.SetEntryPoint (fb, PEFileKinds.WindowApplication);
-			ab.Save (file);
-			return fullFile;
+			tb.CreateTypeInfo();
+			return tb.FullName;
```

Comment:
What about this ?
Will the change do the same. I have no idea what it is.

Reply:
It's basically the path to the executable that would be written by the above code.

That means the code that does the running of the saved assembly:
https://github.com/mono/mono-addins/pull/169/files#diff-c7001386acf361c301b4a1e3d0da4fb27fb5e7d6396ba081583caf2e34b805fbR74

Should be modified to handle this scenario.

Reply:
I have it running and is working in my case. When is this used ?

I'm not sure if that is possible in netstandard !!

I will have to investigate more.

Reply:
Is  there anyone who have an idea, on how to solve this in netstandard ?

Reply:
@mikkeljohnsen one solution would be to make a separate project that contains the emitted code and run it from an assembly on disk that's shipped with Mono.Addins. In example, `Mono.Addins.Setup.exe` could be the one, it calls `SetupProcess.Main`

Reply:
The original code does not seem to work. The generated exe needs to have the Mono.Addins.dll copied into its directory otherwise it does not work.

Also not sure why an exe needs to be generated on the fly here - maybe I am missing something. Seems like a pre-built executable could be used since all the generated code does it call into SetupProcess.Main. Which currently does not work since SetupProcess is an internal class.

Reply:
I had an attempt I did not get to test here:
https://github.com/mono/mono-addins/pull/149

It used IVT to avoid that issue: https://github.com/mono/mono-addins/pull/149/files#diff-1c2789e6e0ffa1098ceee92df92b1714007f0e81372b351d39fa0465eca9737eR26